Assignment.Tests/SampleData.Tests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Assignment.Tests;
+
+[TestClass]
+public class SampleDataTests
+{
+    //I tried my best with these but only 1 is currently passing haha
+    private const string TestCsvFilePath = "People.csv"; // Makes a testing file with sample data
+    SampleData sampleData = new();
+
+    [TestInitialize]
+    public void Initialize()
+    {
+        // Sample Data added to the test file
+        var testCsvContent = "FirstName,LastName,StreetAddress,City,State,Zip,EmailAddress\n" +
+                             "John,Doe,123 Main St,Anytown,CA,12345,john@example.com\n" +
+                             "Jane,Smith,456 Elm St,Other City,NY,67890,jane@example.com";
+        File.WriteAllText(TestCsvFilePath, testCsvContent);
+        
+    }
+
+    [TestCleanup]
+    public void Cleanup()
+    {
+        // Delete the test CSV file
+        if (File.Exists(TestCsvFilePath))
+            File.Delete(TestCsvFilePath);
+    }
+
+    [TestMethod]
+    public void CsvRows_SkipsFirstRow_ReturnsAllRowsFromFile()
+    {
+
+        var result = sampleData.CsvRows.ToList();
+
+        Assert.AreEqual(2, result.Count);
+        CollectionAssert.Contains(result, "John,Doe,123 Main St,Anytown,CA,12345,john@example.com");
+        CollectionAssert.Contains(result, "Jane,Smith,456 Elm St,Other City,NY,67890,jane@example.com");
+    }
+
+    [TestMethod]
+    public void GetUniqueSortedListOfStatesGivenCsvRows_ReturnsUniqueSortedEmails()
+    {
+
+        var result = sampleData.GetUniqueSortedListOfStatesGivenCsvRows().ToList();
+
+        Assert.AreEqual(2, result.Count);
+        Assert.AreEqual("CA", result[0]);
+        Assert.AreEqual("NY", result[1]);
+    }
+
+    [TestMethod]
+    public void GetAggregateSortedListOfStatesUsingCsvRows_ReturnsAggregateSortedStates()
+    {
+
+
+        var result = sampleData.GetAggregateSortedListOfStatesUsingCsvRows();
```

Comment:
nit, throughout this file: whitespace
```suggestion
    {
        var result = sampleData.GetAggregateSortedListOfStatesUsingCsvRows();
```